src/platform/np_spake2.h

```diff
@@ -24,7 +24,6 @@ struct nc_spake2_password_request {
 
 
 struct np_spake2_module {
-    np_error_code (*create)(struct np_platform* pl, struct np_spake2_context** spake);
```

Comment:
Maybe we should remove these spake2 changes and make them in a seperate pull request.